src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj

```diff
@@ -23,10 +23,6 @@
     <Compile Include="System.Runtime.WindowsRuntime.Manual.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ReferencePath Include="$(ExternalAPIsPath)\NetFX\Contracts\DesignTimeFacades\mscorlib.dll" />
```

Comment:
Interesting you are right it isn't referenced. Why is that error not showing up with Dev14 toolset?